2-mandatory/2-retrying-random-numbers.js

```diff
@@ -10,7 +10,11 @@ function generateRandomNumber() {
 }
 
 function getRandomNumberGreaterThan50() {
```

Comment:
This solution works well.

Have you installed Prettier and run it in your VSCode? What does the style guide say about this? https://syllabus.codeyourfuture.io/guides/code-style-guide#using-prettier-to-format-code-automatically